pyiron_contrib/atomistic/atomistics/job/trainingcontainer.py

```diff
@@ -50,6 +52,18 @@ def __init__(self, project, job_name):
             "number_of_atoms": []
         })
 
+    def include_job(self, job, iteration_step=-1):
+        """
+        Add structure, energy and forces from job.
+
+        Args:
+            job (:class:`.AtomisticGenericJob`): job to take structure from
+            iteration_step (int, optional): if job has multiple steps, this selects which to add
+        """
+        self.include_structure(job.get_structure(iteration_step=iteration_step),
+                               energy=job.output.energy_pot[-1], forces=job.output.forces[-1],
```

Comment:
Should not this be iteration_step ? Otherwise it is inconsistent

Reply:
Yes, I missed that.

Reply:
I don't think I can add an easy test for this, though. :/